docs/news.rst

```diff
@@ -4014,3 +4318,4 @@ First release of Scrapy.
 .. _What is cacheable: https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.9.1
```

Comment:
I think three lines above there is a link that it's not working and that could be removed "Twisted - hello, asynchronous programming"